test/tools/yulopti.cpp

```diff
@@ -171,28 +208,138 @@ class YulOpti
 		}
 	}
 
-	void disambiguate()
+	void objectApplyFunction(shared_ptr<Object> _object, function<void(Object&)> _fn)
```

Comment:
```suggestion
	void applyFunctionToObject(shared_ptr<Object> _object, function<void(Object&)> _function)
```
I think we should definitely expand the `fn` abbreviation, and maybe the method name looks better this way?
I guess we could also consider being even more specific and using `applyFunctionToObjectAndSubObjects`... but maybe then we cross the line of readability/simplicity.